docs/contributing.md

```diff
@@ -14,7 +16,64 @@ Nano has a relatively small community of developers and only a handful of develo
 
-### Contribute to the ecosystem
+#### Contribute to the ecosystem
 
 - <a href="https://github.com/mistakia/nano-community/blob/main/CONTRIBUTING.md" target="_blank">Nano.Community</a>: community gateway and knowledge hub
 - <a href="https://github.com/Nault/Nault/blob/master/CONTRIBUTING.md" target="_blank">Nault</a>: a popular community maintained wallet
+
+## Marketing
+
+Before you can effectively contribute to Nano adoption and marketing it is a good idea to have a working understanding of the following:
+
+- [Why it matters](/introduction/why-it-matters)
+- [Advantages](/introduction/advantages)
+- [Investment Thesis](/introduction/investment-thesis)
+- [How it works](/introduction/how-it-works)
+- [Misconceptions](/introduction/misconceptions)
+
+Next, you'll want to understand the vision and roadmap so that we all pull in the same direction.
+
+#### Vision, Roadmap & Goals
+
+Nano aims to be a form of money both replacing existing use cases (payments, remittances, donations, etc) and inventing new previously impossible use cases (micropayments, "streaming" value, etc). Thus, use as a system to transfer value is the focus.
+
+Being that Nano is a protocol, to help increase use of the network, it is important to attract entrepeneurs, developers and enthusiasts to help build and improve it. Nano, much like the internet at first, is still hard to use for the vast majority of consumers and businesses. Thus we need builders to make it more accessible so that everyone can use it.
+
+#### Target Audience
+
+##### Speculators / Holders
+
+Why: For Nano to sustain high real world payment volume it needs to be worth something and have enough liquidity. Thus before there is utility, we need speculators and holders. Due to the positive feedback loop of its network effects, the more who are invested, the more attention the project gets, the more businesses are interested in integrating Nano payments, which then attracts even more investors, companies, developers and exchanges continuing the feedback loop.
+
+Pitch: Nano is a strongly undervalued cryptocurrency. Both as a medium of exchange and store of value, it is fundamentally the best there is in the space. As a medium of exchange, Nano being instant, feeless and scalable means that it is hard to imagine something better. As a store of value, Nano's zero inflation, eco-friendliness and game theory leading to ever more decentralization over time make it potentially the best store of value. However, many cryptocurrencies have higher market caps than Nano. If one analyses any other cryptocurrency that claims to be a store of value or medium of exchange, they'll find that Nano is fundamentally better. Because of this, it stands to reason that as the market gets more mature and rational, Nano's recognition and price will increase.
```

Comment:
Instead (or perhaps in addition to) of stating it is undervalued, it's better to say it is an asymmetric investment. Perhaps stating an estimate for what the potential market cap would be for such a system.